apps/charterafrica/src/payload/fields/slug/formatSlug.js

```diff
@@ -4,15 +4,31 @@ const format = (val) =>
     .replace(/[^\w-]+/g, "")
     .toLowerCase();
 
+const getFallbackData = ({ fallback, originalDoc, data }) => {
+  let fallbackData = "";
+  fallback.forEach((value, i) => {
+    const prefix = i === 0 ? "" : "-";
+    fallbackData += `${prefix}${
+      (data && data[value]) || (originalDoc && originalDoc[value])
+    }`;
+  });
```

Comment:
```suggestion
  const fallbackValues = fallback.map((value) => (
    (data && data[fallback]) || (originalDoc && originalDoc[fallback])
  ));
  return fallbackValues.join("-");
  });
```

Or something like that.